
try :
    print(x)
except:
    print("An error occurred")
finally:
    print("Success")



try :
    num1 = 20
    num2 = 0
    print(num1 / num2)
except :
    print("An error occurred")

